tar/chunks/chunks_write.c

```diff
@@ -284,21 +284,21 @@ chunks_write_printstats(FILE * stream, CHUNKS_W * C, int csv)
 {
 
 	/* Print header. */
-	if (chunks_stats_printheader(stream, csv))
+	if (chunks_stats_printheader(stream, csv, 0))
```

Comment:
Same argument for `-c --null-output`; if we wanted to enable it, this is where we'd start.